orion-commons/src/main/java/com/pinterest/orion/teletraan/TeletraanConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.pinterest.orion.teletraan;
+
+/**
+ * Constants for Teletraan client
+ */
+public class TeletraanConstants {
```

Comment:
nit: Change to `Constants` since `teletraan` is already in the package?

Reply:
Make sense. Changed